cmake/Modules/BuildAngpow.cmake

```diff
@@ -5,12 +5,12 @@ if(${CMAKE_VERSION} VERSION_GREATER "3.10.0")
   set(SHALLOW_GIT_CLONE GIT_SHALLOW 1)
 endif()
 
-set(AngpowTag v0.4.1)
+set(AngpowTag patch-1)
 
 # Downloads and compiles Angpow
 ExternalProject_Add(ANGPOW
         PREFIX ANGPOW
-        GIT_REPOSITORY https://github.com/LSSTDESC/Angpow4CCL.git
+        GIT_REPOSITORY https://github.com/beckermr/Angpow4CCL.git
```

Comment:
Needs to be hacked back to original state once the PR gets merged.

Reply:
This PR https://github.com/LSSTDESC/Angpow4CCL/pull/6 is the one we are blocked on. IDK why the CCL interface is in the upstream library. I would have thought that would be maintained here.

Reply:
This is done!